java/compaction/compaction-job-execution/src/main/java/sleeper/compaction/job/execution/CompactSortedFilesRunner.java

```diff
@@ -94,9 +91,6 @@ public static Builder builder() {
 
     public void run() throws InterruptedException, IOException {
         Instant startTime = Instant.now();
```

Comment:
It looks like this line can be moved into CompactionTask now rather than passing it in?